shopinvader_customer_multi_user/tests/test_multi_user_management.py

```diff
@@ -10,14 +10,14 @@
 class TestUserManagement(TestUserManagementCommmon):
```

Comment:
what to expect on this file:
- Nothing changed, expect `user_bindingN` is renamed to `userN_binding` for consistency